oracledb/README.md

```diff
@@ -0,0 +1,2 @@
+helm install <deployment_name> . --namespace soans --set 'imagePullSecrets[0].name=image-secret'
```

Comment:
This isn't enough documentation. We'd need more, including the correct versions of Helm and container images, etc.

Reply:
reworked the README